config/routes.rb

```diff
@@ -4,12 +4,13 @@
   get 'unauthorised' => 'pages#unauthorised'
   #Users routes
   resources :users do
-    resources :accounts
+    resources :transactions
+    resources :accounts 
   end
-
   #Session routes
   get '/login', to: 'sessions#login'
   post '/login', to: 'sessions#create'
+  get '/logout', to: 'sessions#destroy'
   delete '/logout', to: 'sessions#destroy'
```

Comment:
dont think you need both of these, if the `delete` works then remove `get` and visa versa